plugins/rapid7_insightvm_cloud/plugin.spec.yaml

```diff
@@ -840,6 +840,48 @@ types:
       title: Remediated Vulnerabilities
       type: '[]string'
       required: false
+  exploits:
+    description:
+      title: Description
+      description: A verbose description of the exploit
+      type: string
+      required: true
```

Comment:
Just checking as they are marked required: true but are all these values always returned whenever the exploit item is returned?

Reply:
yes, it always returned these fields